src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrderStaticGroupSeperated.java

```diff
@@ -0,0 +1,10 @@
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
+
+import static com.puppycrawl.tools.checkstyle.internal.XpathUtil.getXpathItems;
```

Comment:
Please try to avoid importing checkstyle into it's unit tests. This creates an unnecessary coupling between the 2 and forces the input to change if the main code changes.

Use something similar to the `junit` static import.

Reply:
Done